core/cont/inc/THashList.h

```diff
@@ -36,7 +36,7 @@ class THashTable;
 class THashList : public TList {
 
 protected:
-   THashTable   *fTable;    //Hashtable used for quick lookup of objects
+   THashTable   *fTable;    //! Hashtable used for quick lookup of objects
```

Comment:
Hi Oliver,

I see the issue ("no programmatic way to distinguish this good case from the "bad", unintended one. ") ... However I still hesitant as 'requiring' both ClassDef(0) *and* //! essentially means that the ClassDef(0) is moot/unnecessary ....

Cheers,
Philippe.